boltdesignsystem/bolt
# Convert Image and Ratio to @bolt/element (LitElement)

## Jira
TBD

## Summary
Converts the Image and Ratio components over to LitElement via our new base Class provided by `@bolt/element`, `BoltElement`.

Early testing shows that this should help considerably with JavaScript execution times / client-side rendering performance!

![1C219D1C-98B7-45C8-885C-0BC846C22D67](https://user-images.githubusercontent.com/1617209/68761522-3816fd80-05e2-11ea-8865-36aaf092a47b.png)
![FA9CD71E-393C-4BC0-BB95-935CF32D093B](https://user-images.githubusercontent.com/1617209/68761523-3816fd80-05e2-11ea-8f73-4bf3f66c9126.png)
![684D1E36-4C8F-4DCE-8839-92DEF3AB274A](https://user-images.githubusercontent.com/1617209/68761524-38af9400-05e2-11ea-9bdd-fe4662d34cfb.png)

 
https://www.webpagetest.org/video/view.php?id=191112_9a75643968a5f66c62c7dd1a396b3038716cad68

## How to test
- Review changes
- Confirm Travis tests all passing as expected 
- Double-check browsers to confirm everything is working as expected (Edge, IE 11, etc)
